assets/application.json

```diff
@@ -27,5 +23,104 @@
     "products/variations/price": []
   },
   "auth_callback_uri": "https://us-central1-ecom-custom-payment.cloudfunctions.net/app/ecom/auth-callback",
-  "version": "1.0.0"
-}
\ No newline at end of file
+  "version": "1.0.0",
+  "admin_settings": {
```

Comment:
O `admin_settings` também deveria estar no ecom.config.js (no objeto `app`), este `assets/application.json` é [gerado automaticamente](https://github.com/ecomplus/application-starter#continuous-integration), da próxima vez que ele for gerado esta alteração sua aqui vai ser sobrescrita.

Reply:
btf